src/web/browser/commercial/init.ts

```diff
@@ -0,0 +1,9 @@
+import '@root/src/web/browser/prepareCmp';
```

Comment:
you may want to add `import '../webpackPublicPath';` to specify the path in case there is the need to do any dynamic imports in the future.